### PR TITLE
[doc]release-numbering-strategies

### PR DESCRIPTION
**Pull request by amber-moe**
As title

### CODE REVIEW

File: doc/release-numbering-strategies.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Descriptions for other semantics:
+
+    Release name | Descriptions
+    --- | ---
+    Pre-Alpha | A pre-release version before Alpha. A pre-Alpha phase is a less complete version.
+    Alpha | A pre-release version.
+    Beta | A pre-release version after Alpha. A beta phase is feature complete but likely to contain a number of known or unknown bugs.
+    RC | A release candidate (RC) version is a beta version likely to be a stable product.
+    BP | A bundle patch version.
```
Comment (longdafeng) [inline doc/release-numbering-strategies.md:25]: Please put GA before BP.

Add notes:  BP version means GA's version with a bundle of patches.

File: doc/release-numbering-strategies.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    GA | A general available version is available for purchase.
```
Comment (longdafeng) [inline doc/release-numbering-strategies.md:26]: Please put GA before BP

File: doc/release-numbering-strategies.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    Stable/Release | A stable or release version passed all the test is for production. The remaining bugs are acceptable.
```
Comment (longdafeng) [inline doc/release-numbering-strategies.md:27]: It seems  stable/release is simlar with GA? it maybe confuse the users. 

Do we need stable/release ?

Comment (Amber1990Zhang) [reply doc/release-numbering-strategies.md:27]: So which one will we pick? GA or Stable/Release? Pls delete  Stable/Release. @LINxiansheng

Comment (Amber1990Zhang) [reply doc/release-numbering-strategies.md:27]: The two are different.

**Review by LINxiansheng — APPROVED** [review]

**Comment by LINxiansheng** [comment]
Thanks for your contribution. 
I found the file was in the wrong folder due to the change is too old.
I change the directory and merge it.

**Pull request closed without merging**